package.json

```diff
@@ -277,7 +278,7 @@
     "webpack": "5.24.2",
     "webpack-bundle-analyzer": "4.7.0",
     "webpack-cli": "^4.2.0",
-    "webpack-dev-server": "3.11.3",
+    "webpack-dev-server": "4.15.1",
```

Comment:
Probably related to webpack upgrade. But now the whole screen is covered in error (before it was only the chrome console). For example when editing a stake i get this 
<img width="1728" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/254a55f8-0b9c-4e68-a3ae-58b620796b22">

Reply:
I think its better that way? before we use to ignore these errors, its better that we resolve issues than getting away from it 😅

Reply:
Yea, i agree, really dont like the console errors we are having. Hopefully this will make us prioritise them more